exporter/redis.go

```diff
@@ -524,6 +537,19 @@ func (e *Exporter) extractInfoMetrics(info, addr string, alias string, scrapes c
 		scrapes <- scrapeResult{Name: metricName, Addr: addr, Alias: alias, Value: val}
 	}
 
+	for dbIndex := 0; dbIndex < dataBasesCount; dbIndex++ {
+
+		dbName := "db" + strconv.Itoa(dbIndex)
+
+		_, exists := handledDBs[dbName]
```

Comment:
you can combine this with line 546

`if _, exists := handledDBs[dbName]; !exists {`

Reply:
combined